adoc/admin-updates.adoc

```diff
@@ -265,7 +265,13 @@ with the `kured` reboot daemon.
 === Disabling Automatic Updates
 
 Nodes added to a cluster have the service `skuba-update.timer`, which is responsible for running automatic updates, activated by default.
+
 This service calls the `skuba-update` utility and it can be configured with the `/etc/sysconfig/skuba-update` file.
+[NOTE]
+===
+`skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches` when running. `--non-interactive-include-reboot-patches` when used in conjunction with `--non-interactive` this will treat patches with the `rebootSuggested-flag` as non-interactive as the reboot will be managed by `kured`.
```

Comment:
You could drop the word "this" in "this will treat patches" as the subject was just stated and it reads fine without the "this".
I might rephrase "as non-interactive as the reboot will be managed by `kured`" to "as non-interactive allowing the reboot to be managed by `kured`".  Having "as" used twice in the sentence for two slightly different meanings just read a little funny in my mind.  But it is ok either way.

Reply:
please read it now and let me know what you think. @cmurphy 's commit corrected the problem I think.